erc20-price/proto/v1/erc20_price.proto

```diff
@@ -2,10 +2,17 @@ syntax = "proto3";
 
 package messari.erc20_price.v1;
 
+message Token {
+  string address = 1;
+  string symbol = 2;
+  uint64 decimals = 3;
+}
+
 message Erc20Price {
-  string token_address = 1;
+  Token token = 1;
```

Comment:
why not just use `Erc20Token`?
https://github.com/messari/substreams/blob/90e50e21abd5288f720aaf3d8f3feeba3a4d01bf/common/proto/erc20.proto#L9

Reply:
Yeah, good point. Let me do that